docs/transactions.md

```diff
@@ -1247,64 +1247,56 @@ Updates the Validator node by the owner.
 - Who can send:
   - NodeAdmin; owner
 
-#### REMOVE_VALIDATOR_NODE
+#### DISABLE_VALIDATOR_NODE
 
-**Status: Not Implemented**
+**Status: Implemented**
 
-Deletes the Validator node (removes from the validator set) by the owner.
+Disables the Validator node (removes from the validator set) by the owner.
 
-- Parameters:
-  - address: `string` - Bech32 encoded validator address or owner account
 - Who can send:
   - NodeAdmin; owner
 
```

Comment:
Please add CLI and REST commands for all new functions

Reply:
Done! ✅